api/valor_api/backend/metrics/detection.py

```diff
@@ -969,85 +1015,13 @@ def _aggregate_data(
     return (groundtruths_cte, predictions_cte, labels)
 
 
-def _compute_detection_metrics(
+@profiler
```

Comment:
should all the profiler references be removed?

Reply:
yes, i just hadnt pushed this commit yet